modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -7,44 +7,52 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Blacklist
->s|Summary      | Blocks a list or range of IP addresses
+>s|Summary      | Blocks access from a range of IP addresses to an API endpoint
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s|First Mule version available  | 3.8.0
 .1+>.^s| Returned Status Codes
 | 403 - IP is rejected
 |===
 
-The IP blacklist policy can control whether a list or range of IP addresses can access and interact with a configured API endpoint. You use one of the following options to blacklist addresses for your API:
+The IP Blacklist policy controls access to a configured API endpoint from a range of IP addresses. You can blacklist access based on the IP address of the request
+or based on the origin IP address of the `x-Forwarded-For` header.
 
-* Option 1: Blacklist access based on the IP address of the request
-* Option 2: Blacklist access based on the origin IP address of the x-Forwarded-For header
+//Nauel--will users know what this header is?
```

Comment:
The `x-forwarded-for` HTTP header is a well known header used by many Internet routers, so the route taken by a message can be audited for example (More info in: https://en.wikipedia.org/wiki/X-Forwarded-For).

Reply:
Done! Thanks!